sw/applications/spi_host_example/spi_host_example.c

```diff
@@ -36,10 +39,28 @@ void handler_irq_fast_spi(void)
     spi_intr_flag = 1;
 }
 
+void handler_irq_fast_spi_flash(void)
+{
+    // Disable SPI interrupts
+    spi_enable_evt_intr(&spi_host, false);
+    spi_enable_rxwm_intr(&spi_host, false);
+
+    // Clear fast interrupt
+    fast_intr_ctrl_t fast_intr_ctrl;
+    fast_intr_ctrl.base_addr = mmio_region_from_addr((uintptr_t)FAST_INTR_CTRL_START_ADDRESS);
+    clear_fast_interrupt(&fast_intr_ctrl, kSpiFlash);
+
+    spi_intr_flag = 1;
+}
```

Comment:
Put it in an ifndef with USE_SPI_FLASH and the handler_irq_fast_spi above function